posthog/hogql_queries/query_runner.py

```diff
@@ -808,6 +814,29 @@ def run(
 
         return fresh_response
 
+    def get_api_queries_concurrency_limit(self):
+        """
+        :return: None - no feature, 0 - rate limited, 1,3,<other> for actual concurrency limit
+        """
+
+        if not settings.EE_AVAILABLE:
+            return None
+
+        from ee.billing.quota_limiting import list_limited_team_attributes, QuotaLimitingCaches, QuotaResource
+
+        if self.team.api_token in list_limited_team_attributes(
+            QuotaResource.API_QUERIES, QuotaLimitingCaches.QUOTA_LIMITER_CACHE_KEY
+        ):
+            return 0
+
+        feature = next(
+            filter(
+                lambda f: f.get("key") == "api_queries_concurrency", self.team.organization.available_product_features
```

Comment:
We don't follow this well but we should add this to the enum `AvailableFeature` in constants.py so it can be reused.